types/leaflet/index.d.ts

```diff
@@ -1615,7 +1615,7 @@ export interface FitBoundsOptions extends ZoomOptions, PanOptions {
     maxZoom?: number | undefined;
 }
 
-export interface PanInsideOptions {
+export interface PanInsideOptions extends ZoomOptions, PanOptions {
```

Comment:
```suggestion
export interface PanInsideOptions extends PanOptions {
```

panInside calls panTo which calls setView which takes pan and zoom options.

However, panTo does not pass through the zoom options: https://github.com/Leaflet/Leaflet/blob/d52d2fa341af4c120262b24a12592cfc3f66020e/src/map/Map.js#L308